File: perfect square.c
int main()
{
    int i,n;
    printf("Enter the number: \n");
    scanf("%d",&n);
    for(i=0;i<=n;i++){
        if(n==i*i){
            printf("Entered number is a perfect number\n");
            return 0;
        }
    }
        printf("Entered number is not perfect number\n");
}

File: PerfectSquare.java
import java.util.*;
class PerfectSquare
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        int n,r,s;
        System.out.println("Enter the number: ");
        n=sc.nextInt();
        r=(int)Math.sqrt(n);
        s=r*r;
        if(s==n)
            System.out.println("its PerfectSquare");
        else
            System.out.println("its not PerfectSquare");
    }
}
